src/components/ReorderInput/ReorderInput.tsx

```diff
@@ -20,26 +20,27 @@ type ReorderInputProps = {
 // User might change title to e.g. bejzik8 and he might forget what is the type of link.
 // I can easily update this if necessary.
 
+const validateInput = (type, content) => {
+  let isValid = false;
+
+  try {
+    const linkValidity = nimiLinkValidator.isValidSync({
```

Comment:
I think there's `schema.isValid` that returns a Boolean.

Reply:
`Schema.isValid` is asynchronous operation and in our case I didn't see a reason to use it over synchronous approach. Is there anything that I'm missing?

Reply:
I opted for the asynchronous because it's the default: https://github.com/jquense/yup#schemavalidatesyncvalue-any-options-object-infertypeschema